README.md

```diff
@@ -150,6 +150,12 @@ try {
  
 **Important: encrypted tokens are intended for single use only. You must not invoke the client using the same token more than once.**
 
+### Working with the ActivityDetails object
+
+The ActivityDetails object contains a UserProfile object containing the details of the user that the app requested, and which can be accessed using a set of helper methods `getUserProfile().getSelfie()`, `getUserProfile().getGender()` etc
```

Comment:
"...requested, **and which** can be accessed...". Is this text copied from somewhere else? Is there a JIRA Issue linked to this?

Reply:
No, feel free to format it however you desire.

Reply:
It was just the removal of and/or in the sentence that I was looking for - wanted to make sure that if it was copied from somewhere else, then we would change it there too!